static/js/util/util.js

```diff
@@ -249,6 +249,6 @@ export function calculateDegreeInclusions(profile: Profile) {
 /**
  * Calls an array of functions in series with a given argument and returns an array of the results
  */
-export function callFunctionArray(functionArray, arg) {
+export function callFunctionArray<T,R>(functionArray: Array<(t: T) => R>, arg: any): R[] {
```

Comment:
@noisecapella this annotation type checks but I'm not sure it's quite correct, since this means that the `callFunctionArray` has type polymorphism, but every function passed in in the `functionArray` has the same argument and return types...